addons/game.controller.ps/layout.xml

```diff
@@ -7,19 +7,24 @@
 		<button name="triangle" type="digital" label="30004"/>
 		<button name="start" type="digital" label="30005"/>
 		<button name="select" type="digital" label="30006"/>
-		<button name="up" type="digital" label="30007" direction="up"/>
-		<button name="down" type="digital" label="30008" direction="down"/>
-		<button name="right" type="digital" label="30009" direction="right"/>
-		<button name="left" type="digital" label="30010" direction="left"/>
-		<button name="l3" type="digital" label="30015"/>
-		<button name="r3" type="digital" label="30016"/>
+		<button name="guide" type="digital" label="30007"
```

Comment:
I think analog is a hardware button that modifies the bahavior of the controller. Not sure though, we should look through the libretro.cpp file of our psx emulators to see if the guide button maps to anything